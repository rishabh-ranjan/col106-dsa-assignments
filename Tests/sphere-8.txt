ADD_TRIANGLE -1.7363184667829046e-08 0.7071066498756409 -0.7071067690849304 0.2705979645252228 0.2705981135368347 -0.9238795042037964 -2.4620234739813895e-09 0.3826834261417389 -0.9238795042037964
ADD_TRIANGLE -4.716550705552436e-08 0.9238793253898621 -0.38268351554870605 0.7071066498756409 0.7071068286895752 -4.371138828673793e-08 0.6532813310623169 0.6532815098762512 -0.38268351554870605
ADD_TRIANGLE -4.716550705552436e-08 0.9238793253898621 0.3826834261417389 0.49999985098838806 0.5 0.7071067690849304 0.6532813310623169 0.6532815098762512 0.3826834261417389
ADD_TRIANGLE -2.4620234739813895e-09 0.3826834261417389 0.9238795042037964 -1.0677015183091498e-07 4.422565069717166e-08 1.0 0.2705979347229004 0.27059808373451233 0.9238795042037964
ADD_TRIANGLE 0.0 1.5099580252808664e-07 -1.0 -2.4620234739813895e-09 0.3826834261417389 -0.9238795042037964 0.2705979645252228 0.2705981135368347 -0.9238795042037964
ADD_TRIANGLE -4.716550705552436e-08 0.9238793253898621 -0.38268351554870605 0.49999985098838806 0.5 -0.7071067690849304 -1.7363184667829046e-08 0.7071066498756409 -0.7071067690849304
ADD_TRIANGLE -4.716550705552436e-08 0.9238793253898621 0.3826834261417389 0.7071066498756409 0.7071068286895752 -4.371138828673793e-08 0.0 1.0 -4.371138828673793e-08
ADD_TRIANGLE -1.7363184667829046e-08 0.7071066498756409 0.7071067690849304 0.2705979347229004 0.27059808373451233 0.9238795042037964 0.49999985098838806 0.5 0.7071067690849304
ADD_TRIANGLE 0.2705979347229004 0.27059808373451233 0.9238795042037964 -1.0677015183091498e-07 4.422565069717166e-08 1.0 0.38268327713012695 1.485337861595326e-07 0.9238795042037964
ADD_TRIANGLE 0.0 1.5099580252808664e-07 -1.0 0.2705979645252228 0.2705981135368347 -0.9238795042037964 0.38268327713012695 1.485337861595326e-07 -0.9238795042037964
ADD_TRIANGLE 0.6532813310623169 0.6532815098762512 -0.38268351554870605 0.7071065306663513 1.6343494735338027e-07 -0.7071067690849304 0.49999985098838806 0.5 -0.7071067690849304
ADD_TRIANGLE 0.6532813310623169 0.6532815098762512 0.3826834261417389 0.9999997615814209 1.3363262496568495e-07 -4.371138828673793e-08 0.7071066498756409 0.7071068286895752 -4.371138828673793e-08
ADD_TRIANGLE 0.2705979347229004 0.27059808373451233 0.9238795042037964 0.7071065306663513 1.6343494735338027e-07 0.7071067690849304 0.49999985098838806 0.5 0.7071067690849304
ADD_TRIANGLE 0.2705979645252228 0.2705981135368347 -0.9238795042037964 0.7071065306663513 1.6343494735338027e-07 -0.7071067690849304 0.38268327713012695 1.485337861595326e-07 -0.9238795042037964
ADD_TRIANGLE 0.7071066498756409 0.7071068286895752 -4.371138828673793e-08 0.9238793253898621 1.6343494735338027e-07 -0.38268351554870605 0.6532813310623169 0.6532815098762512 -0.38268351554870605
ADD_TRIANGLE 0.49999985098838806 0.5 0.7071067690849304 0.9238793253898621 1.6343494735338027e-07 0.3826834261417389 0.6532813310623169 0.6532815098762512 0.3826834261417389
ADD_TRIANGLE 0.9238793253898621 1.6343494735338027e-07 -0.38268351554870605 0.4999998211860657 -0.49999967217445374 -0.7071067690849304 0.7071065306663513 1.6343494735338027e-07 -0.7071067690849304
ADD_TRIANGLE 0.9238793253898621 1.6343494735338027e-07 0.3826834261417389 0.7071065902709961 -0.7071064114570618 -4.371138828673793e-08 0.9999997615814209 1.3363262496568495e-07 -4.371138828673793e-08
ADD_TRIANGLE 0.38268327713012695 1.485337861595326e-07 0.9238795042037964 0.4999998211860657 -0.49999967217445374 0.7071067690849304 0.7071065306663513 1.6343494735338027e-07 0.7071067690849304
ADD_TRIANGLE 0.7071065306663513 1.6343494735338027e-07 -0.7071067690849304 0.2705979347229004 -0.27059778571128845 -0.9238795042037964 0.38268327713012695 1.485337861595326e-07 -0.9238795042037964
ADD_TRIANGLE 0.9999997615814209 1.3363262496568495e-07 -4.371138828673793e-08 0.6532813310623169 -0.6532811522483826 -0.38268351554870605 0.9238793253898621 1.6343494735338027e-07 -0.38268351554870605
ADD_TRIANGLE 0.7071065306663513 1.6343494735338027e-07 0.7071067690849304 0.6532813310623169 -0.6532811522483826 0.3826834261417389 0.9238793253898621 1.6343494735338027e-07 0.3826834261417389
ADD_TRIANGLE 0.38268327713012695 1.485337861595326e-07 0.9238795042037964 -1.0677015183091498e-07 4.422565069717166e-08 1.0 0.2705979347229004 -0.27059778571128845 0.9238795042037964
ADD_TRIANGLE 0.0 1.5099580252808664e-07 -1.0 0.38268327713012695 1.485337861595326e-07 -0.9238795042037964 0.2705979347229004 -0.27059778571128845 -0.9238795042037964
ADD_TRIANGLE 0.4999998211860657 -0.49999967217445374 -0.7071067690849304 1.2439137719866267e-08 -0.382683128118515 -0.9238795042037964 0.2705979347229004 -0.27059778571128845 -0.9238795042037964
ADD_TRIANGLE 0.7071065902709961 -0.7071064114570618 -4.371138828673793e-08 4.224146010756158e-08 -0.923879086971283 -0.38268351554870605 0.6532813310623169 -0.6532811522483826 -0.38268351554870605
ADD_TRIANGLE 0.4999998211860657 -0.49999967217445374 0.7071067690849304 4.224146010756158e-08 -0.923879086971283 0.3826834261417389 0.6532813310623169 -0.6532811522483826 0.3826834261417389
ADD_TRIANGLE 0.2705979347229004 -0.27059778571128845 0.9238795042037964 -1.0677015183091498e-07 4.422565069717166e-08 1.0 1.2439137719866267e-08 -0.382683128118515 0.9238795042037964
ADD_TRIANGLE 0.0 1.5099580252808664e-07 -1.0 0.2705979347229004 -0.27059778571128845 -0.9238795042037964 1.2439137719866267e-08 -0.382683128118515 -0.9238795042037964
ADD_TRIANGLE 0.4999998211860657 -0.49999967217445374 -0.7071067690849304 4.224146010756158e-08 -0.923879086971283 -0.38268351554870605 1.2439137719866267e-08 -0.707106351852417 -0.7071067690849304
ADD_TRIANGLE 0.6532813310623169 -0.6532811522483826 0.3826834261417389 1.2439137719866267e-08 -0.9999995231628418 -4.371138828673793e-08 0.7071065902709961 -0.7071064114570618 -4.371138828673793e-08
ADD_TRIANGLE 0.2705979347229004 -0.27059778571128845 0.9238795042037964 1.2439137719866267e-08 -0.707106351852417 0.7071067690849304 0.4999998211860657 -0.49999967217445374 0.7071067690849304
ADD_TRIANGLE 1.2439137719866267e-08 -0.9999995231628418 -4.371138828673793e-08 -0.6532812118530273 -0.6532811522483826 -0.38268351554870605 4.224146010756158e-08 -0.923879086971283 -0.38268351554870605
ADD_TRIANGLE 1.2439137719866267e-08 -0.707106351852417 0.7071067690849304 -0.6532812118530273 -0.6532811522483826 0.3826834261417389 4.224146010756158e-08 -0.923879086971283 0.3826834261417389
ADD_TRIANGLE 1.2439137719866267e-08 -0.382683128118515 0.9238795042037964 -1.0677015183091498e-07 4.422565069717166e-08 1.0 -0.2705979347229004 -0.27059778571128845 0.9238795042037964
ADD_TRIANGLE 0.0 1.5099580252808664e-07 -1.0 1.2439137719866267e-08 -0.382683128118515 -0.9238795042037964 -0.2705979347229004 -0.27059778571128845 -0.9238795042037964
ADD_TRIANGLE 1.2439137719866267e-08 -0.707106351852417 -0.7071067690849304 -0.6532812118530273 -0.6532811522483826 -0.38268351554870605 -0.4999998211860657 -0.49999967217445374 -0.7071067690849304
ADD_TRIANGLE 1.2439137719866267e-08 -0.9999995231628418 -4.371138828673793e-08 -0.6532812118530273 -0.6532811522483826 0.3826834261417389 -0.7071065306663513 -0.707106351852417 -4.371138828673793e-08
ADD_TRIANGLE 1.2439137719866267e-08 -0.707106351852417 0.7071067690849304 -0.2705979347229004 -0.27059778571128845 0.9238795042037964 -0.4999998211860657 -0.49999967217445374 0.7071067690849304
ADD_TRIANGLE 1.2439137719866267e-08 -0.707106351852417 -0.7071067690849304 -0.2705979347229004 -0.27059778571128845 -0.9238795042037964 1.2439137719866267e-08 -0.382683128118515 -0.9238795042037964
ADD_TRIANGLE 0.0 1.5099580252808664e-07 -1.0 -0.2705979347229004 -0.27059778571128845 -0.9238795042037964 -0.38268327713012695 1.6343494735338027e-07 -0.9238795042037964
ADD_TRIANGLE -0.6532812118530273 -0.6532811522483826 -0.38268351554870605 -0.7071065306663513 1.6343494735338027e-07 -0.7071067690849304 -0.4999998211860657 -0.49999967217445374 -0.7071067690849304
ADD_TRIANGLE -0.7071065306663513 -0.707106351852417 -4.371138828673793e-08 -0.9238792061805725 1.0383029547256228e-07 0.3826834261417389 -0.9999996423721313 1.6343494735338027e-07 -4.371138828673793e-08
ADD_TRIANGLE -0.2705979347229004 -0.27059778571128845 0.9238795042037964 -0.7071065306663513 1.6343494735338027e-07 0.7071067690849304 -0.4999998211860657 -0.49999967217445374 0.7071067690849304
ADD_TRIANGLE -0.4999998211860657 -0.49999967217445374 -0.7071067690849304 -0.38268327713012695 1.6343494735338027e-07 -0.9238795042037964 -0.2705979347229004 -0.27059778571128845 -0.9238795042037964
ADD_TRIANGLE -0.7071065306663513 -0.707106351852417 -4.371138828673793e-08 -0.9238792061805725 1.0383029547256228e-07 -0.38268351554870605 -0.6532812118530273 -0.6532811522483826 -0.38268351554870605
ADD_TRIANGLE -0.4999998211860657 -0.49999967217445374 0.7071067690849304 -0.9238792061805725 1.0383029547256228e-07 0.3826834261417389 -0.6532812118530273 -0.6532811522483826 0.3826834261417389
ADD_TRIANGLE -0.2705979347229004 -0.27059778571128845 0.9238795042037964 -1.0677015183091498e-07 4.422565069717166e-08 1.0 -0.38268327713012695 1.6343494735338027e-07 0.9238795042037964
ADD_TRIANGLE -0.9999996423721313 1.6343494735338027e-07 -4.371138828673793e-08 -0.6532812714576721 0.6532813310623169 0.3826834261417389 -0.7071065306663513 0.7071067094802856 -4.371138828673793e-08
ADD_TRIANGLE -0.38268327713012695 1.6343494735338027e-07 0.9238795042037964 -0.4999998211860657 0.4999999701976776 0.7071067690849304 -0.7071065306663513 1.6343494735338027e-07 0.7071067690849304
ADD_TRIANGLE -0.38268327713012695 1.6343494735338027e-07 -0.9238795042037964 -0.4999998211860657 0.4999999701976776 -0.7071067690849304 -0.2705979347229004 0.27059808373451233 -0.9238795042037964
ADD_TRIANGLE -0.9999996423721313 1.6343494735338027e-07 -4.371138828673793e-08 -0.6532812714576721 0.6532813310623169 -0.38268351554870605 -0.9238792061805725 1.0383029547256228e-07 -0.38268351554870605
ADD_TRIANGLE -0.7071065306663513 1.6343494735338027e-07 0.7071067690849304 -0.6532812714576721 0.6532813310623169 0.3826834261417389 -0.9238792061805725 1.0383029547256228e-07 0.3826834261417389
ADD_TRIANGLE -0.38268327713012695 1.6343494735338027e-07 0.9238795042037964 -1.0677015183091498e-07 4.422565069717166e-08 1.0 -0.2705979347229004 0.27059808373451233 0.9238795042037964
ADD_TRIANGLE 0.0 1.5099580252808664e-07 -1.0 -0.38268327713012695 1.6343494735338027e-07 -0.9238795042037964 -0.2705979347229004 0.27059808373451233 -0.9238795042037964
ADD_TRIANGLE -0.7071065306663513 1.6343494735338027e-07 -0.7071067690849304 -0.6532812714576721 0.6532813310623169 -0.38268351554870605 -0.4999998211860657 0.4999999701976776 -0.7071067690849304
ADD_TRIANGLE -0.4999998211860657 0.4999999701976776 -0.7071067690849304 -2.4620234739813895e-09 0.3826834261417389 -0.9238795042037964 -0.2705979347229004 0.27059808373451233 -0.9238795042037964
ADD_TRIANGLE -0.7071065306663513 0.7071067094802856 -4.371138828673793e-08 -4.716550705552436e-08 0.9238793253898621 -0.38268351554870605 -0.6532812714576721 0.6532813310623169 -0.38268351554870605
ADD_TRIANGLE -0.4999998211860657 0.4999999701976776 0.7071067690849304 -4.716550705552436e-08 0.9238793253898621 0.3826834261417389 -0.6532812714576721 0.6532813310623169 0.3826834261417389
ADD_TRIANGLE -0.2705979347229004 0.27059808373451233 0.9238795042037964 -1.0677015183091498e-07 4.422565069717166e-08 1.0 -2.4620234739813895e-09 0.3826834261417389 0.9238795042037964
ADD_TRIANGLE 0.0 1.5099580252808664e-07 -1.0 -0.2705979347229004 0.27059808373451233 -0.9238795042037964 -2.4620234739813895e-09 0.3826834261417389 -0.9238795042037964
ADD_TRIANGLE -0.6532812714576721 0.6532813310623169 -0.38268351554870605 -1.7363184667829046e-08 0.7071066498756409 -0.7071067690849304 -0.4999998211860657 0.4999999701976776 -0.7071067690849304
ADD_TRIANGLE -0.6532812714576721 0.6532813310623169 0.3826834261417389 0.0 1.0 -4.371138828673793e-08 -0.7071065306663513 0.7071067094802856 -4.371138828673793e-08
ADD_TRIANGLE -0.2705979347229004 0.27059808373451233 0.9238795042037964 -1.7363184667829046e-08 0.7071066498756409 0.7071067690849304 -0.4999998211860657 0.4999999701976776 0.7071067690849304
ADD_TRIANGLE -1.7363184667829046e-08 0.7071066498756409 -0.7071067690849304 0.49999985098838806 0.5 -0.7071067690849304 0.2705979645252228 0.2705981135368347 -0.9238795042037964
ADD_TRIANGLE -4.716550705552436e-08 0.9238793253898621 -0.38268351554870605 0.0 1.0 -4.371138828673793e-08 0.7071066498756409 0.7071068286895752 -4.371138828673793e-08
ADD_TRIANGLE -4.716550705552436e-08 0.9238793253898621 0.3826834261417389 -1.7363184667829046e-08 0.7071066498756409 0.7071067690849304 0.49999985098838806 0.5 0.7071067690849304
ADD_TRIANGLE -4.716550705552436e-08 0.9238793253898621 -0.38268351554870605 0.6532813310623169 0.6532815098762512 -0.38268351554870605 0.49999985098838806 0.5 -0.7071067690849304
ADD_TRIANGLE -4.716550705552436e-08 0.9238793253898621 0.3826834261417389 0.6532813310623169 0.6532815098762512 0.3826834261417389 0.7071066498756409 0.7071068286895752 -4.371138828673793e-08
ADD_TRIANGLE -1.7363184667829046e-08 0.7071066498756409 0.7071067690849304 -2.4620234739813895e-09 0.3826834261417389 0.9238795042037964 0.2705979347229004 0.27059808373451233 0.9238795042037964
ADD_TRIANGLE 0.6532813310623169 0.6532815098762512 -0.38268351554870605 0.9238793253898621 1.6343494735338027e-07 -0.38268351554870605 0.7071065306663513 1.6343494735338027e-07 -0.7071067690849304
ADD_TRIANGLE 0.6532813310623169 0.6532815098762512 0.3826834261417389 0.9238793253898621 1.6343494735338027e-07 0.3826834261417389 0.9999997615814209 1.3363262496568495e-07 -4.371138828673793e-08
ADD_TRIANGLE 0.2705979347229004 0.27059808373451233 0.9238795042037964 0.38268327713012695 1.485337861595326e-07 0.9238795042037964 0.7071065306663513 1.6343494735338027e-07 0.7071067690849304
ADD_TRIANGLE 0.2705979645252228 0.2705981135368347 -0.9238795042037964 0.49999985098838806 0.5 -0.7071067690849304 0.7071065306663513 1.6343494735338027e-07 -0.7071067690849304
ADD_TRIANGLE 0.7071066498756409 0.7071068286895752 -4.371138828673793e-08 0.9999997615814209 1.3363262496568495e-07 -4.371138828673793e-08 0.9238793253898621 1.6343494735338027e-07 -0.38268351554870605
ADD_TRIANGLE 0.49999985098838806 0.5 0.7071067690849304 0.7071065306663513 1.6343494735338027e-07 0.7071067690849304 0.9238793253898621 1.6343494735338027e-07 0.3826834261417389
ADD_TRIANGLE 0.9238793253898621 1.6343494735338027e-07 -0.38268351554870605 0.6532813310623169 -0.6532811522483826 -0.38268351554870605 0.4999998211860657 -0.49999967217445374 -0.7071067690849304
ADD_TRIANGLE 0.9238793253898621 1.6343494735338027e-07 0.3826834261417389 0.6532813310623169 -0.6532811522483826 0.3826834261417389 0.7071065902709961 -0.7071064114570618 -4.371138828673793e-08
ADD_TRIANGLE 0.38268327713012695 1.485337861595326e-07 0.9238795042037964 0.2705979347229004 -0.27059778571128845 0.9238795042037964 0.4999998211860657 -0.49999967217445374 0.7071067690849304
ADD_TRIANGLE 0.7071065306663513 1.6343494735338027e-07 -0.7071067690849304 0.4999998211860657 -0.49999967217445374 -0.7071067690849304 0.2705979347229004 -0.27059778571128845 -0.9238795042037964
ADD_TRIANGLE 0.9999997615814209 1.3363262496568495e-07 -4.371138828673793e-08 0.7071065902709961 -0.7071064114570618 -4.371138828673793e-08 0.6532813310623169 -0.6532811522483826 -0.38268351554870605
ADD_TRIANGLE 0.7071065306663513 1.6343494735338027e-07 0.7071067690849304 0.4999998211860657 -0.49999967217445374 0.7071067690849304 0.6532813310623169 -0.6532811522483826 0.3826834261417389
ADD_TRIANGLE 0.4999998211860657 -0.49999967217445374 -0.7071067690849304 1.2439137719866267e-08 -0.707106351852417 -0.7071067690849304 1.2439137719866267e-08 -0.382683128118515 -0.9238795042037964
ADD_TRIANGLE 0.7071065902709961 -0.7071064114570618 -4.371138828673793e-08 1.2439137719866267e-08 -0.9999995231628418 -4.371138828673793e-08 4.224146010756158e-08 -0.923879086971283 -0.38268351554870605
ADD_TRIANGLE 0.4999998211860657 -0.49999967217445374 0.7071067690849304 1.2439137719866267e-08 -0.707106351852417 0.7071067690849304 4.224146010756158e-08 -0.923879086971283 0.3826834261417389
ADD_TRIANGLE 0.4999998211860657 -0.49999967217445374 -0.7071067690849304 0.6532813310623169 -0.6532811522483826 -0.38268351554870605 4.224146010756158e-08 -0.923879086971283 -0.38268351554870605
ADD_TRIANGLE 0.6532813310623169 -0.6532811522483826 0.3826834261417389 4.224146010756158e-08 -0.923879086971283 0.3826834261417389 1.2439137719866267e-08 -0.9999995231628418 -4.371138828673793e-08
ADD_TRIANGLE 0.2705979347229004 -0.27059778571128845 0.9238795042037964 1.2439137719866267e-08 -0.382683128118515 0.9238795042037964 1.2439137719866267e-08 -0.707106351852417 0.7071067690849304
ADD_TRIANGLE 1.2439137719866267e-08 -0.9999995231628418 -4.371138828673793e-08 -0.7071065306663513 -0.707106351852417 -4.371138828673793e-08 -0.6532812118530273 -0.6532811522483826 -0.38268351554870605
ADD_TRIANGLE 1.2439137719866267e-08 -0.707106351852417 0.7071067690849304 -0.4999998211860657 -0.49999967217445374 0.7071067690849304 -0.6532812118530273 -0.6532811522483826 0.3826834261417389
ADD_TRIANGLE 1.2439137719866267e-08 -0.707106351852417 -0.7071067690849304 4.224146010756158e-08 -0.923879086971283 -0.38268351554870605 -0.6532812118530273 -0.6532811522483826 -0.38268351554870605
ADD_TRIANGLE 1.2439137719866267e-08 -0.9999995231628418 -4.371138828673793e-08 4.224146010756158e-08 -0.923879086971283 0.3826834261417389 -0.6532812118530273 -0.6532811522483826 0.3826834261417389
ADD_TRIANGLE 1.2439137719866267e-08 -0.707106351852417 0.7071067690849304 1.2439137719866267e-08 -0.382683128118515 0.9238795042037964 -0.2705979347229004 -0.27059778571128845 0.9238795042037964
ADD_TRIANGLE 1.2439137719866267e-08 -0.707106351852417 -0.7071067690849304 -0.4999998211860657 -0.49999967217445374 -0.7071067690849304 -0.2705979347229004 -0.27059778571128845 -0.9238795042037964
ADD_TRIANGLE -0.6532812118530273 -0.6532811522483826 -0.38268351554870605 -0.9238792061805725 1.0383029547256228e-07 -0.38268351554870605 -0.7071065306663513 1.6343494735338027e-07 -0.7071067690849304
ADD_TRIANGLE -0.7071065306663513 -0.707106351852417 -4.371138828673793e-08 -0.6532812118530273 -0.6532811522483826 0.3826834261417389 -0.9238792061805725 1.0383029547256228e-07 0.3826834261417389
ADD_TRIANGLE -0.2705979347229004 -0.27059778571128845 0.9238795042037964 -0.38268327713012695 1.6343494735338027e-07 0.9238795042037964 -0.7071065306663513 1.6343494735338027e-07 0.7071067690849304
ADD_TRIANGLE -0.4999998211860657 -0.49999967217445374 -0.7071067690849304 -0.7071065306663513 1.6343494735338027e-07 -0.7071067690849304 -0.38268327713012695 1.6343494735338027e-07 -0.9238795042037964
ADD_TRIANGLE -0.7071065306663513 -0.707106351852417 -4.371138828673793e-08 -0.9999996423721313 1.6343494735338027e-07 -4.371138828673793e-08 -0.9238792061805725 1.0383029547256228e-07 -0.38268351554870605
ADD_TRIANGLE -0.4999998211860657 -0.49999967217445374 0.7071067690849304 -0.7071065306663513 1.6343494735338027e-07 0.7071067690849304 -0.9238792061805725 1.0383029547256228e-07 0.3826834261417389
ADD_TRIANGLE -0.9999996423721313 1.6343494735338027e-07 -4.371138828673793e-08 -0.9238792061805725 1.0383029547256228e-07 0.3826834261417389 -0.6532812714576721 0.6532813310623169 0.3826834261417389
ADD_TRIANGLE -0.38268327713012695 1.6343494735338027e-07 0.9238795042037964 -0.2705979347229004 0.27059808373451233 0.9238795042037964 -0.4999998211860657 0.4999999701976776 0.7071067690849304
ADD_TRIANGLE -0.38268327713012695 1.6343494735338027e-07 -0.9238795042037964 -0.7071065306663513 1.6343494735338027e-07 -0.7071067690849304 -0.4999998211860657 0.4999999701976776 -0.7071067690849304
ADD_TRIANGLE -0.9999996423721313 1.6343494735338027e-07 -4.371138828673793e-08 -0.7071065306663513 0.7071067094802856 -4.371138828673793e-08 -0.6532812714576721 0.6532813310623169 -0.38268351554870605
ADD_TRIANGLE -0.7071065306663513 1.6343494735338027e-07 0.7071067690849304 -0.4999998211860657 0.4999999701976776 0.7071067690849304 -0.6532812714576721 0.6532813310623169 0.3826834261417389
ADD_TRIANGLE -0.7071065306663513 1.6343494735338027e-07 -0.7071067690849304 -0.9238792061805725 1.0383029547256228e-07 -0.38268351554870605 -0.6532812714576721 0.6532813310623169 -0.38268351554870605
ADD_TRIANGLE -0.4999998211860657 0.4999999701976776 -0.7071067690849304 -1.7363184667829046e-08 0.7071066498756409 -0.7071067690849304 -2.4620234739813895e-09 0.3826834261417389 -0.9238795042037964
ADD_TRIANGLE -0.7071065306663513 0.7071067094802856 -4.371138828673793e-08 0.0 1.0 -4.371138828673793e-08 -4.716550705552436e-08 0.9238793253898621 -0.38268351554870605
ADD_TRIANGLE -0.4999998211860657 0.4999999701976776 0.7071067690849304 -1.7363184667829046e-08 0.7071066498756409 0.7071067690849304 -4.716550705552436e-08 0.9238793253898621 0.3826834261417389
ADD_TRIANGLE -0.6532812714576721 0.6532813310623169 -0.38268351554870605 -4.716550705552436e-08 0.9238793253898621 -0.38268351554870605 -1.7363184667829046e-08 0.7071066498756409 -0.7071067690849304
ADD_TRIANGLE -0.6532812714576721 0.6532813310623169 0.3826834261417389 -4.716550705552436e-08 0.9238793253898621 0.3826834261417389 0.0 1.0 -4.371138828673793e-08
ADD_TRIANGLE -0.2705979347229004 0.27059808373451233 0.9238795042037964 -2.4620234739813895e-09 0.3826834261417389 0.9238795042037964 -1.7363184667829046e-08 0.7071066498756409 0.7071067690849304
COUNT_COMPONENTS
TYPE_MESH
CENTROID
MAXIMUM_DIAMETER