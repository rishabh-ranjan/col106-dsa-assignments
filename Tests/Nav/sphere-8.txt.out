true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
true
1
[(2.3636325E-8, 1.1355713E-7, -1.7470327E-8)]
15
